libs/prelude/Prelude.idr

```diff
@@ -37,6 +37,8 @@ import public Language.Reflection.Errors
 %default total
 
 -- Things that can't be elsewhere for import cycle reasons
+%runElab (defineFunction $ DefineFun `{void} [])
```

Comment:
Hmm, another possible option is to move the definition of `void` to Uninhabited and have Uninhabited import Nat rather than the other way around. Testing...